reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
 New content
 -----------
 
-While contributing new content, it is easier to work with the docs on your local machine. You can submit a PR after all the checks have passed and things looks satisfactory. You'll need ``make`` and ``python3`` installed on your system.
+When adding new content, it's easier to work with the documentation on your local machine. Once you've made your changes, ensure all checks have passed and everything looks satisfactory before submitting a pull request (PR). To run the checks, you must have ``make`` and ``python3`` installed on your system.
+
+Use the `Diátaxis`_ approach to create new categories for this documentation. These categories can have subcategories within them as well. The documentation has a base ``index.rst`` file, which is usually the landing page. The ``index.rst`` file contains the documentation title, content introducing the documentation and its purpose, relevant links, and a toctree. The toctree lists the categories contained in the documentation in the order they appear in the navigation. Each category and subcategory also contains an ``index.rst`` file similar to the documentation's base ``index.rst`` file.
+
+The documentation structure should look like this:
+
+.. code-block:: none
+
+   documentation/
+   ├── tutorial/
+   │   ├── subcategory-one/
+   │   │   ├── index.rst
+   │   │   ├── page-one.rst
+   │   │   ├── page-two.rst
+   │   │   └── page-three.rst
+   │   └── subcategory-two/
+   │   |   ├── index.rst
+   │   |   ├── page-one.rst
+   │   |   ├── page-two.rst
+   │   |   └── page-three.rst
+   |   └── index.rst
+   ├── how-to-guides
+   ├── explanation
+   ├── reference
+   └── index.rst
+
+Create new categories
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create new categories to the documentation by following these steps:
+
+1. Create a new folder in your documentation directory.
+2. Create a new ``index.rst`` file within your new folder.
+3. Add the title of your new category to the first line of the ``index.rst`` file. Underline it using equal signs (=) that match the length of your title. For more information on titles and headings, read the `reStructuredText`_ style guide.
+4. In the ``index.rst`` file, add content introducing the category, its purpose, and other relevant links.
+5. In your ``index.rst`` file, add a toctree that specifies the file names of pages and the index files of the subcategories within your newly created category. The toctree should resemble the following structure:
+
+.. code:: rst
+
+   .. toctree::
+      :maxdepth: 2
+
+      subcategory-one/index
+      Subcategory two <subcategory-two/index>
+      page-one-file-name
+
+For more information, read the `Sphinx documentation on toctree`_.
+
+5. Update the documentation base ``index.rst`` file by adding your new category to the toctree.
```

Comment:
Maybe change it to something like "Update the project's main ``index.rst`` file by adding your new category to its toctree."